common/transaction.go

```diff
@@ -0,0 +1,13 @@
+package common
```

Comment:
why common? it should be a part of broadcast interfaces. 
I mean all:
- interfaces with methods
- input models
- output models
- errors
should be in the same package because they're a bundle that is suppose to be some contract for any "broadcasting" thing in this library

Reply:
moved to a broadcast package